rockchip: stop "heartbeat" LED for Radxa CM3I(E25)

RadxaNaoki opened the pull request:
stop "heartbeat" which happens before OpenWrt controls LED. instead, just turn LED on.

1715173329 commented:
https://github.com/openwrt/openwrt/blob/main/target/linux/rockchip/patches-6.6/112-radxa-e25-add-led-aliases.patch doesn't work?

RadxaNaoki commented:
> https://github.com/openwrt/openwrt/blob/main/target/linux/rockchip/patches-6.6/112-radxa-e25-add-led-aliases.patch doesn't work?

it's working. in addition to them, heatbeat is working at initial boot stage(before openwrt control), so I disable it.

1715173329 commented:
thanks for clarification. I would suggest merging this into 112-radxa-e25-add-led-aliases.patch

RadxaNaoki committed:
rockchip: stop "heartbeat" LED for Radxa CM3I(E25)

stop "heartbeat" which happens before OpenWrt controls LED. instead,
just turn LED on.

Signed-off-by: FUKAUMI Naoki <naoki@radxa.com>
Link: https://github.com/openwrt/openwrt/pull/15913
Signed-off-by: Robert Marko <robimarko@gmail.com>

RadxaNaoki commented:
> thanks for clarification. I would suggest merging this into 112-radxa-e25-add-led-aliases.patch

done

robimarko commented:
Thanks! Rebased on top of main and merged!
